texthero/preprocessing.py

```diff
@@ -680,12 +708,12 @@ def tokenize(s: pd.Series) -> pd.Series:
 
     """
 
-    punct = string.punctuation.replace("_", "")
     # In regex, the metacharacter 'w' is "a-z, A-Z, 0-9, including the _ (underscore) character." We therefore remove it from the punctuation string as this is already included in \w
+    punct = string.punctuation.replace("_", "")
```

Comment:
As far as I can see, we can just do at the top (where you define the other patterns):
```
punct = string.punctuation.replace("_", "")
TOKENIZATION = rf"((\w)([{punct}])(?:\B|$)|(?:^|\B)([{punct}])(\w))"
```
and just use the `TOKENIZATION` pattern here and delete the `_get_pattern_for_tokenization` function to have prettier code.